packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -141,9 +140,14 @@ export default class ContentSwitcher extends React.Component {
       ...other
     } = this.props;
 
+    const isIconOnly = React.Children.map(children, (child) => {
+      return child.type.displayName === 'IconSwitch';
```

Comment:
Just want to make a note here that this locks the API so consumers can only use IconSwitch. It could prove problematic if someone were to to roll their own IconSwitch component and names it something different. Let's cross that bridge if we come to it though - no need to over optimize for it right now imo